WORKSPACE

```diff
@@ -37,17 +37,24 @@ load("//repositories:deps.bzl", container_deps = "deps")
 
 container_deps()
 
-# pip deps are only needed for running  tests.
+# pip deps are only needed for running tests.
 load("//repositories:pip_repositories.bzl", "pip_deps")
 
 pip_deps()
 
+load(
+    "//repositories:images.bzl",
+    rules_docker_images = "images",
+)
+
+rules_docker_images()
```

Comment:
move this call to def deps in //repositories:deps.bzl

Reply:
Done